pkg/aspect/metricsManager.go

```diff
@@ -55,7 +55,7 @@ func newMetricsManager() ReportManager {
 	return &metricsManager{}
 }
 
-func (m *metricsManager) NewReportExecutor(c *cpb.Combined, a adapter.Builder, env adapter.Env, df descriptor.Finder) (ReportExecutor, error) {
+func (m *metricsManager) NewReportExecutor(c *cpb.Combined, createAdapater CreateAspectFunc, env adapter.Env, df descriptor.Finder) (ReportExecutor, error) {
```

Comment:
I always get confused, this is basically "CreateAdapter" right ? Can we rename the CreateAspectFunc to CreateAdapterFunc